websites/es/examples/outline.md

```diff
@@ -0,0 +1,214 @@
+
+ - `outline` funciona de forma similar al método `headings` mencionado anteriormente. También devuelve todos los encabezados, pero mantiene la estructura del documento original en su lugar y proporciona los niveles de encabezado (por ejemplo, `h1`) solo con la salida.
+
+ - El método `outlineWithParagraphs` funciona de forma similar a `outline`, con la diferencia de que esta llamada también incluye los párrafos.
+
+ - CleanOutlineWithParagraphs` funciona de forma similar a `outlineWithParagraphs`, con la diferencia de que se eliminan las etiquetas HTML vacías.
+
+Los siguientes ejemplos deberían ayudar a entender mejor la funcionalidad. Hay métodos dedicados para la [extracción de palabras clave](/es/examples/extract-keywords) disponibles.
+
+
+## Extraer el esquema
+
+El esquema del contenido permite construir un índice del documento. El siguiente ejemplo construye una versión markdown de los encabezados del documento solicitado:
+
+```php
+$web = new \spekulatius\phpscraper();
+
+/**
+ * Navegue hasta la página de la prueba. Esta página contiene:
+ *
+ * <h1>¡Estamos probando aquí!</h1>
+ * [...]
+ *
+ * <h2>Ejemplos</h2>
+ * [...]
+ *
+ * <h3>Ejemplo 1</h3>
+ * [...]
+ *
+ * <h3>Ejemplo 2</h3>
+ * [...]
+ *
+ * <h3>Ejemplo 3</h3>
+ * [...]
+ */
+$web->go('https://test-pages.phpscraper.de/content/outline.html');
+
+/**
+ * $outline se establecerá para que contenga:
+ *
+ * [
+ *    [
+ *      "tag" => "h1",
+ *      "content" =>  "¡Estamos probando aquí!"
+ *    ], [
+ *      "tag" => "h2",
+ *      "content" => "Ejemplos"
+ *    ], [
+ *      "tag" => "h3",
+ *      "content" => "Ejemplo 1"
+ *    ], [
+ *      "tag" => "h3",
+ *      "content" => "Ejemplo 2"
+ *    ], [
+ *      "tag" => "h3",
+ *      "content" => "Ejemplo 3"
+ *    ]
+ * ]
+ */
+$outline = $web->outline;
+```
+
+
+## Extraer el esquema con párrafos
+
+El siguiente método funciona de manera similar a `outline`, pero también incluye los párrafos como parte del array devuelto:
+
+```php
+$web = new \spekulatius\phpscraper();
+
+/**
+ * Navegue hasta la página de la prueba. Esta página contiene:
+ *
+ * <h1>¡Aquí estamos probando!</h1>
+ * <p>Esta página contiene una estructura de ejemplo para ser analizada. Viene con una serie de encabezados y párrafos anidados como ejemplo de scrape.</p>
+ *
+ * <h2>Ejemplos</h2>
+ * <p>Hay numerosos ejemplos en el sitio web. Por favor, compruébalos para obtener más contexto sobre cómo funciona el scraping.</p> <p>
+ *
+ * <h3>Ejemplo 1</h3>
+ * <p>Aquí hay un ejemplo.</p> <p>
+ *
+ * <h3>Ejemplo 2</h3>
+ * <p>Aquí estaría el segundo ejemplo.</p> <p>
+ *
+ * <h3>Ejemplo 3</h3>
+ * <p>Aquí estaría otro ejemplo.</p> <p>
+ *
+
+ * <p></p>
+ */
+$web->go('https://test-pages.phpscraper.de/content/outline.html');
+
+
+$content = $web->outlineWithParagraphs;
+/**
+ * $content ahora contiene:
+ *
+ * [
```

Comment:
Can you replace this print out with the English version in DE/ES?

Reply:
Yes, I am also looking for all such places where we need to use English only 👍🏼